tests/mbt/model/README.md

```diff
@@ -94,6 +106,13 @@ with a timestamp >= t + UnbondingPeriod on that consumer.
 - [X] EventuallyMatureOnProviderInv: If we send a VscPacket, this is eventually responded to by all consumers
 that were running at the time the packet was sent (and are still running).
 
+Invariants only relevant when running with key assignment (`--step stepKeyAssignment`):
+- [X] ValidatorSetHasExistedKeyAssignmentInv: Replaces ValidatorSetHasExistedInv. Validator sets are checked for equality under key assignment when checking whether they have existed.
```

Comment:
nit: States that it replaces `ValidatorSetHasExistedInv` but `ValidatorSetHasExistedInv` seems to still exist.

Reply:
it is meant to replace ValidatorSetHasExistedInv when running with --step stepKeyAssignment
I will make this clearer